clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -1148,7 +1228,7 @@ private void handleResponses(long now, List<ClientResponse> responses) {
                     } catch (Throwable t) {
                         if (log.isTraceEnabled())
                             log.trace("{} handleResponse failed with {}", call, prettyPrintException(t));
-                        call.fail(now, t);
+                        call.retryOrFail(call, t, now);
```

Comment:
Is there a reason that line 1218 has replaced `call.fail` with `call.retry` but line 1231 has replaced `call.fail` with `call.retryOrFail`?